csharp/src/Google.Protobuf.Test/JsonFormatterTest.cs

```diff
@@ -300,7 +319,8 @@ public void WrapperFormatting_IncludeNull()
         {
             // The actual JSON here is very large because there are lots of fields. Just test a couple of them.
             var message = new TestWellKnownTypes { Int32Field = 10 };
-            var formatter = new JsonFormatter(new JsonFormatter.Settings(true));
+            var formatter = new JsonFormatter(new JsonFormatter.Settings(
```

Comment:
Are you making these changes just for the sake of clarity, using a named argument? I think it's still using the obsolete constructor.